demisto_sdk/commands/init/initiator.py

```diff
@@ -627,3 +635,28 @@ def copy_demistotmock(self):
                 shutil.copy(f'{self.configuration.env_dir}/Tests/demistomock/demistomock.py', self.full_output_path)
             except Exception as err:
                 print_v(f'Could not copy demistomock: {str(err)}')
+
+    def get_remote_templates(self, files_list):
+        """
+        Downloading the object related template-files and saving them in the output path.
+        Args:
+            files_list: List of files to download.
+        Returns:
+            bool. True if the files were downloaded and saved successfully, False otherwise.
+        """
+        if self.is_integration:
+            path = os.path.join('Packs', 'HelloWorld', 'Integrations', 'HelloWorld')
+            os.mkdir(os.path.join(self.full_output_path, self.TEST_DATA_DIR))
+        else:
+            path = os.path.join('Packs', 'HelloWorld', 'Scripts', 'HelloWorldScript')
+
+        for file in files_list:
+            try:
+                file_content = tools.get_remote_file(os.path.join(path, file), return_content=True)
+                with open(os.path.join(self.full_output_path, file), 'wb') as f:
+                    f.write(file_content)
+            except Exception:
+                print(f"Could not fetch remote template - {file}. Using local templates instead.")
```

Comment:
also, specify the error message
```suggestion
                print_warning(f"Could not fetch remote template - {file}. Using local templates instead.")
```

Reply:
the relevant exception is being printed as part of `get_remote_file` function.
`get_remote_file` returns an empty dict if an error has occurred so the error that the above function raises would be `a bytes-like object is required, not 'dict'`.
Should I add it anyway